pkg/minikube/cruntime/cri.go

```diff
@@ -40,8 +40,20 @@ type container struct {
 func crictlList(cr CommandRunner, root string, o ListContainersOptions) (*command.RunResult, error) {
 	klog.Infof("listing CRI containers in root %s: %+v", root, o)
 
+	var filter string
+	switch o.State {
+	case All:
+		filter = "-a"
+	case Running, Paused:
+		// crictl does not understand paused containers
+		// we have to list all the running ones and later filter out paused ones by runc
```

Comment:
mind clarifying where we do the filtering out paused ones? is this required for this PR or is it a good to have ?

Reply:
Thanks! It's done with `runc` later:

https://github.com/kubernetes/minikube/blob/e6ea64792f1521f5444897171d4a852ecfaea910/pkg/minikube/cruntime/cri.go#L134-L137

This is just a nice to have to avoid getting all containers when we are only looking for `paused` onces.

The original logic:
1. Get all containers from `crictl`
2. Filter paused containers by `runc`

New logic:
1. Get running containers from `crictl` (Paused containers will have `Running` state in crictl)
2. Filter paused containers by `runc`